client-libraries/rust/modality-reflector-config/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "modality-reflector-config"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+modality-ingest-client = { version = "0.1", path = "../modality-ingest-client" }
```

Comment:
It's probably a bit much to pull in the entire ingest crate just for `AttrVal` representation and parsing, especially since it has relatively heavy dependencies on `tokio` and the TLS stuff.  A quick fix would be to make the modality-ingest-client crate have those dependencies be optional, tied to a "client" feature and conditionally compile the `client` module based on that feature. Said feature could be part of the default feature-set, but `modality-reflector-config` could opt out of the default features.

Reply:
I wondered about that.

I justified doing this to myself by assuming that anything is going to be run by the reflector is also going to talk to the reflector and would essentially need this library anyway.

Is that actually a reasonable assumption?

Reply:
Config inspection or editing tooling might want to read config without running an ingest client.

Separately, the mutation protocol configuration is also managed in the reflector config.  When the mutation story advances further, folks will probably want to be able to read the config for that side of the world without pulling in the ingest client guts also.